### ISSUE TITLE
No sound in some flash clips, and no errors in the F12 console

### ISSUE THREAD

**Issue by Diatonator** [ruffle-rs/ruffle#2307]
No sound in the flash clips:

https://z0r.de/166
https://z0r.de/1875

The issue https://github.com/ruffle-rs/ruffle/issues/2074 I logged about the absent sound, and where it appears to be because of not-yet-implemented Nellymoser support, shows this error:
![image](https://user-images.githubusercontent.com/75184268/103819133-deda1b00-507a-11eb-994e-8e5df3c653cf.png)

**Comment by Diatonator** [comment]
Nellymoser codec got merged into Ruffle, but those flashes still have no sound, thus looks like not the Nellymoser problem

**Comment by Diatonator** [comment]
The issue persists on Nightly 2021-12-30

**Comment by n0samu** [comment]
166 has the same issue as #1820: 
```as
_root.onLoad = function()
{
   hun = new Sound();
   hun.attachSound("hurr");
   hun.start(0,999);
};
```
_root.onLoad never fires in Ruffle, so the sound never starts.

1875 has a similar issue to #2435. Here is the code on frame 1 of the movie:
```as
if(_root.getBytesLoaded() >= _root.getBytesTotal())
{
   gotoAndPlay(3);
}
```
The StartSound tag is on frame 2. So if the movie gets to frame 1 and bytesLoaded = bytesTotal, frame 2 will be skipped and the sound will never start. Since Ruffle loads all bytes of the movie before beginning playback, that's exactly what happens. In Flash Player, adding trace statements before the aforementioned code shows that playback begins before all bytes are loaded, and thus frame 2 is not skipped:
```as
trace(_root.getBytesLoaded()); //traces 66506
trace(_root.getBytesTotal()); //traces 679759
```
Note: getBytesLoaded traces the same value every time! Interesting that this is deterministic...

**Comment by n0samu** [comment]
The problem with z0r 1875 is also mentioned in #6144.
